frontend/packages/dev-console/integration-tests/features/pipelines/pipelines-Actions.feature

```diff
@@ -68,34 +68,34 @@ Examples:
 
 @regression, @smoke
 Scenario Outline: Edit the Pipeline from pipelines Details page : P-08-TC01
-   Given user is at the pipelines page
+   Given user is at pipelines page
    And pipeline with name "<pipeline_name>" is present on Pipelines page
-   When user searches with pipeline name "<pipeline_name>" on Pipelines page
+   When the user enters "<pipeline_name>" into the pipelines search bar
    And click pipeline name "<pipeline_name>" from searched results on Pipelines page
    And user selects the option "Edit Pipeline" from Actions menu drop down
    Then user is at the Pipeline Builder page
-   And "<pipeline_name>" field should be disabled
-   And "Add Parameters" link, "Add Resources" link, "Task"  should be enabled
+   And Name field should be disabled
```

Comment:
```suggestion
   And user will see Name field disabled
```

Reply:
Will update it as "Name filed will be disabled"